src/docfx/lib/HtmlUtility.cs

```diff
@@ -16,25 +15,27 @@ internal static class HtmlUtility
         private static readonly Func<HtmlAgilityPack.HtmlAttribute, int> s_getValueStartIndex =
             ReflectionUtility.CreateInstanceFieldGetter<HtmlAgilityPack.HtmlAttribute, int>("_valuestartindex");
 
-        public static HtmlNode LoadHtml(string html)
-        {
-            var doc = new HtmlDocument();
-            doc.LoadHtml(html);
-            return doc.DocumentNode;
-        }
+        public static string HtmlPostProcess(string html, CultureInfo culture) => HtmlPostProcess(LoadHtml(html), culture);
 
-        public static string TransformHtml(string html, Func<HtmlNode, HtmlNode> transform)
+        public static string HtmlPostProcess(HtmlNode html, CultureInfo culture)
```

Comment:
What's the meaning of PostProcess for an utility?  Seems it's belongs to BuildPage html post process, not HtmlUtility.
We should name it using functional description for utility method, right？

Reply:
Suggestion to move this method to BuildPage

Reply:
It will be shared with conceptual and html/markdown format processing (attribute/schematransform) in SDP, thus extracted to HtmlUtil. The named was picked and agreed with the template team to replace logics in conceptual jint, in case they want to apply some post processing inside build pipeline after we generate HTML.

Reply:
this name has no difference with `DoSomeThings` in HtmlUtility, for me, it's more like a business logic combination.